supervisor/supervisor.go

```diff
@@ -274,11 +279,17 @@ func (s *Supervisor) Start() error {
 		"memory":      s.machine.Memory,
 		"cpus":        s.machine.Cpus,
 	}).Debug("containerd: supervisor running")
-	go func() {
-		for i := range s.tasks {
-			s.handleTask(i)
-		}
-	}()
+
+	wg := &sync.WaitGroup{}
+	for i := 0; i < GetDefaultWorkersNum(); i++ {
```

Comment:
Because of the reason like https://github.com/containerd/containerd/pull/1177,  ... s.containers[t.ID] need serialization , close this pr ,thank you.